oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,249 @@
+facilitating communication amongst community members, `selecting new members`_,
+and ensuring that existing members renew their commitment to the community and
+abide by the Code of Conduct. Program administrators are empowered to monitor
+Core Contributor conduct, particularly whether they follow the "3 Cs"
+(Commitment, Conduct, Caliber), and may issue reminders, suspensions, and even
+removal from the Program. Program administrators can be contacted at
+``cc-program-admins@tcril.org``.
+
+The tCRIL administrator(s) are the point people for making sure that needed
+agreements are signed - this includes a Program Agreement between the CC and
+tCRIL (sent to you by a Program Administrator); for code contributors, an
+additional CLA (signed when you make your first PR); and for new entities
+joining the community, an Entity Contributor Agreement (please communicate with
+a Program Administrator about this).
+
+We are additionally asking that all employers of CCs sign the `Revised
+Declaration of Commitment to the Core Contributor Program
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/3216900524>`_ to
+demonstrate that they are willing to engage with and commit resources to the
+Open edX project. We encourage a commitment of 20 hours/month from each of an
+organization's Core Contributors, however, we recognize that circumstances
+differ and a different commitment may be more appropriate. Before signing the
+agreement, please reach out to the Program Administrators to discuss.
+
+Questions about the Program can be directed to the tCRIL administrator(s) via
+the ``#core-contributors`` Slack room in the `Open edX Slack
+<https://openedx.slack.com/>`_.
+
+.. _selecting new members:
+
+Adding New Core Contributors
+----------------------------
+
+New Core Contributors are determined via a nomination process - CCs may nominate
+new members, or existing community members may put themselves forward for a role.
+
+Current CCs should always be on the lookout for contributors who demonstrate the
+"3 Cs" (Commitment, Conduct, and Caliber), follow the `Code of Conduct
+<https://open.edx.org/code-of-conduct/>`_, are respectful of other community members, are polite and
+welcoming to all in the community, and have been an overall active community member for some time.
+When a person like this comes to the attention of a CC, the CC should reach out and ask if
+that person would like to be nominated as a CC.
+
+Community members who feel they fit these criteria listed above should feel empowered to
+nominate themselves, as well.
+
+Note that when a new Core Contributor role is being defined and piloted, however,
+nominations should proceed only once the Program Administrator is confident that new Core
+Contributors in that role can be supported. Reach out to the Administrator when there is
+any uncertainty around the status of the role; Administrators can be contacted at ``cc-program-admins@tcril.org``.
+
```

Comment:
Re @nedbat 's [comment](https://github.com/openedx/open-edx-proposals/pull/275#issuecomment-1022619102):

> Maybe it's implied here, but I think at any given time the Program Administrators will be able to say, "we have enough CCs right now, nominations are closed." True? If so, do we want to state it?

Hmm.. that's worth mentioning here, though I hope we don't ever have to invoke it! It's like saying "sorry, we're too busy to hire new people right now" :) But how about:

```suggestion
At times, the Core Contributor program may suspend accepting new members, when there is insufficient support for additional people in a given role. Program Administrators will indicate this by posting in the `Core Contributors discussion category <https://discuss.openedx.org/c/working-groups/core-contributors/36>`_.
```